IPs/RSKIP200.md

```diff
@@ -0,0 +1,65 @@
+# Bridge performance improvement
+
+|RSKIP          |200           |
+| :------------ |:-------------|
+|**Title**      |Receive headers limits |
+|**Created**    |08-JAN-21 |
+|**Author**     |PGP |
+|**Purpose**    |Sca,USa,Sec |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+
+## Abstract
+
+This RSKIP proposes to include new functionality in the bridge precompiled contract to receive one header every time that this function is called and save this header in the internal storage of the bridge. 
+This also included a modification in the actual method to receive headers in the same precompiled contract. These methods receive an array of headers to be stored. The modification for this method implies not allow to be called for external users after this RSKIP. 
+
+## Motivation
+
+Bridge precompiled contract includes a function to can receive an array of heads to be stored. This function could be exposed to attacks. To avoid this, some new limits are included. 
+Powpeg nodes need to send a group of headers anyway. 
+Since one of these new limits is to receive only one header per time, and these limits are needed only for external users, a new method with the new limits is created and the original one was modified to be used only by powpeg nodes. 
```

Comment:
```suggestion
Since powpeg nodes still need to send a group of headers to the Bridge, the current `receiveHeaders` function is made private. A new function `receiveHeader` is created, which only allows registering one header at a time, to be used by external users.
```